hw/chisel/src/main/scala/snax/readerWriter/Reader.scala

```diff
@@ -57,7 +55,7 @@ class Reader(param: ReaderWriterParam, clusterName: String = "unnamed_cluster")
       outputWidth = param.tcdmParam.dataWidth * param.tcdmParam.numChannel,
       depth = param.bufferDepth
     ) {
-      override val desiredName = s"${clusterName}_xdma_Reader_DataBuffer"
+      override val desiredName = s"${clusterName}_Reader_DataBuffer"
```

Comment:
This is some form of camelcase + snake case? Can we keep consistent spelling in future?

Reply:
When Scala converting to Verilog, the naming convention will also with “_” and this is unavoidable. Here is the guidance of naming in Verilog, so as you see the “_” appears😂